### PR TITLE
Split query and skeleton

### PR DESCRIPTION
**Pull request by antho1404**
Improve loaders in all pages with independent queries and skeletons

- https://github.com/liteflow-labs/starter-kit/pull/256
- https://github.com/liteflow-labs/starter-kit/pull/265
- https://github.com/liteflow-labs/starter-kit/pull/267
- https://github.com/liteflow-labs/starter-kit/pull/268
- https://github.com/liteflow-labs/starter-kit/pull/269
- https://github.com/liteflow-labs/starter-kit/pull/270
- https://github.com/liteflow-labs/starter-kit/pull/271
- https://github.com/liteflow-labs/starter-kit/pull/272
- https://github.com/liteflow-labs/starter-kit/pull/274

### CODE REVIEW

**Comment by antho1404** [comment]
@ismailToyran could you have a quick review, you can check all the PRs independently

File: pages/tokens/[id]/index.tsx [hunk 1]
```diff
@@ -230,14 +203,13 @@ const DetailPage: NextPage<Props> = ({ now: nowProp }) => {
...
-  if (loading) return <Loader fullPage />
-  if (!asset) return <></>
+  if (!loading && !asset) return <Error statusCode={404} />
   return (
     <LargeLayout>
       <Head
-        title={asset.name}
-        description={asset.description}
-        image={asset.image}
+        title={asset?.name || 'loading...'}
```
Comment (ismailToyran) [inline pages/tokens/[id]/index.tsx:210]: Translate file for `loading...` needed?

**Review by ismailToyran — APPROVED** [review]
Some comments, but looking great code wise.
- Didn't check design wise (how skeletons look)
- Didn't check query wise (if all the unnecessary ones has been removed or not)

File: components/HomeSection/Assets.tsx [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+          aggregates {
+            distinctCount {
+              currencyId
+            }
+          }
+        }
+      }
+    }
+  }
+`
```
[the remark this answers is not in the document]
Comment (antho1404) [reply components/HomeSection/Assets.tsx:160]: why is that? this was actually moved from the index page to this component so the component is now responsible to load the data

File: components/HomeSection/Auctions.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                decimals
+                symbol
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`
```
[the remark this answers is not in the document]
Comment (antho1404) [reply components/HomeSection/Auctions.tsx:91]: similar here

File: pages/checkout/[id].tsx [hunk 4]
```diff
@@ -50,16 +53,24 @@ const CheckoutPage: NextPage<Props> = ({ now }) => {
...
+    return [
+      offer.asset.chainId,
+      offer.asset.collectionAddress,
+      offer.asset.tokenId,
+    ].join('-')
+  }, [offer])
+
+  const assetQuery = useFetchAssetForCheckoutQuery({
+    variables: { now: date, address: address || '', id: assetId || '' },
+    skip: !assetId,
```
[the remark this answers is not in the document]
Comment (antho1404) [reply pages/checkout/[id].tsx:70]: Address is not required to load the data as it is only to check connected users and possibly know or not if the user is owner. Also it's completely possible to have an empty address if the user is not connected

File: pages/tokens/[id]/index.tsx (hunk 1 above)
Comment (antho1404) [reply pages/tokens/[id]/index.tsx]: just removed like all the other pages

File: components/HomeSection/Assets.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ismailToyran) [reply components/HomeSection/Assets.tsx:160]: We already have this `query` on `HomeSection/Assets.gql`

File: components/HomeSection/Auctions.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (ismailToyran) [reply components/HomeSection/Auctions.tsx:91]: We already have this `query` on `HomeSection/Auctions.gql`

File: components/HomeSection/Assets.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (antho1404) [reply components/HomeSection/Assets.tsx]: ok my bad it was in both the tsx and gql, probably an issue with some merge at some point
Removed now

File: components/HomeSection/Auctions.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (antho1404) [reply components/HomeSection/Auctions.tsx]: removed

**Comment by antho1404** [comment]
@EmmanuelDrouin please feel free to have a look UI-wise if some skeleton need to be improved and create issues if that's the case

**Pull request merged**